lua/tardis/screens/cl_classic_screensaver.lua

```diff
@@ -0,0 +1,34 @@
+-- currently disabled due to visual gui implementation
```

Comment:
I don't think this needs to return, I think this can just be deleted

Reply:
For now, I'm keeping it as a template. 
Will do as soon as the new screensaver is added